### PR TITLE
Update elastic-agent-libs to v0.9.11

### PR DESCRIPTION
**Pull request by belimawr**
## What does this PR do?

Update elastic-agent-libs to v0.9.11

## Why is it important?

It updates elastic-agent-libs to the latest version. The previous version had a bug, even though the Elastic-Agent was not affected by it, it's better to keep elastic-agent-libs udated

## Checklist

- [ ] ~~My code follows the style guidelines of this project~~
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [ ] ~~I have made corresponding change to the default configuration files~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~I have added an entry in `./changelog/fragments` using the [changelog tool](https://github.com/elastic/elastic-agent#changelog)~~
- [ ] ~~I have added an integration test or an E2E test~~

## Disruptive User Impact
There is no user inpact

## How to test this PR locally
1. Build the PR
2. Run elastic-agent in standalone mode
3. Assert that logs go to stdout and `data/elastic-agent-<hash>/logs`

~~## Related issues~~

## Questions to ask yourself

- How are we going to support this in production?
- How are we going to measure its adoption?
- How are we going to debug this?
- What are the metrics I should take care of?
- ...

### CODE REVIEW

**Comment by belimawr** [comment]
Converting into draft until https://github.com/elastic/elastic-agent-libs/pull/209 gets merged and I can update this PR to use v0.9.10

File: go.mod [hunk 1]
```diff
@@ -16,7 +16,7 @@ require (
 	github.com/elastic/e2e-testing v1.2.1
 	github.com/elastic/elastic-agent-autodiscover v0.6.14
 	github.com/elastic/elastic-agent-client/v7 v7.9.0
-	github.com/elastic/elastic-agent-libs v0.9.8
```
[the remark this answers is not in the document]
Comment (belimawr) [reply go.mod:19]: Yes, done  by bafc343785

**Comment by belimawr** [comment]
buildkite test this

**Review by kaanyalti — APPROVED** [review]

**Comment by belimawr** [comment]
I found another issue, back to draft until https://github.com/elastic/elastic-agent-libs/pull/210 gets merged

**Pull request merged**